config/asgi.py

```diff
@@ -11,6 +11,6 @@
 
 from django.core.asgi import get_asgi_application
 
-os.environ.setdefault("DJANGO_SETTINGS_MODULE", "config.settings")
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "config.settings.production")
```

Comment:
Shouldn't we be able to read this variable from where it's set in the `docker-compose` file so we don't have to get it right here?

Reply:
We should read the environment variable but have a fallback to production if an environment variable is missing. It comes by default when creating a Django app